docs/aws.md

```diff
@@ -148,8 +141,42 @@ subdomain.kubernetes.com.        172800  IN  NS  ns-1022.awsdns-35.com.
 subdomain.kubernetes.com.        172800  IN  NS  ns-1149.awsdns-27.co.uk.
 ```
 
-## Setting up a state store for your cluster
+## Create an IAM user for kops
+In this example we will be using a dedicated IAM user to use with kops. This user will need basic API security credentials in order to use kops. Create the user and credentials using the AWS console. [More information](https://aws.amazon.com/documentation/iam/)
+
+The kops user will require the following IAM permission to function properly:
+```
+AmazonEC2FullAccess
+AmazonRoute53FullAccess
+AmazonS3FullAccess
+IAMFullAccess
+AmazonVPCFullAccess
+```
+
+You can create a kops IAM user from the command line using the following:
 
+```bash
+aws iam create-group --group-name kops
+
+export arns="
+arn:aws:iam::aws:policy/AmazonEC2FullAccess
+arn:aws:iam::aws:policy/AmazonRoute53FullAccess
+arn:aws:iam::aws:policy/AmazonS3FullAccess
+arn:aws:iam::aws:policy/IAMFullAccess
+arn:aws:iam::aws:policy/AmazonVPCFullAccess"
+
+for arn in $arns; do aws iam attach-group-policy --policy-arn "$arn" --group-name kops; done
+
+aws iam create-user --user-name testuser
+
+aws iam add-user-to-group --user-name kops --group-name kops
+
+aws iam create-access-key --user-name kops
+# make note of the SecretAccesKey and AccessKeyID in the returned json output
+```
+Kubernetes kops uses the official AWS Go SDK, so all we need to do here is set up your system to use the official AWS supported methods of registering security credentials defined [here](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html#specifying-credentials). Follow the example below to configure the aws command line tool to set up the security credentials for the kops user you just created.
```

Comment:
you are right, that line should have been removed when I moved the kops IAM user section to it's new location. Just pushed a change to fix that.